tests/testthat/test-clean_fusen_files.R

```diff
@@ -111,12 +115,13 @@ test_that("df_to_config fails when appropriate", {
 # Create files, even empty
 dir_tmp <- tempfile()
 dir.create(dir_tmp)
-file.create(file.path(dir_tmp, c("zaza.R", "zozo.R", "test-zaza.R")))
+file.create(file.path(dir_tmp, c("zaza.R", "zozo.R", "test-zaza.R", "toto.Rmd")))
```

Comment:
I see that you add "toto.Rmd".  
What do you expect to see or not to see with this file in the unit tests ?

Reply:
This file was deleted, as it was not useful anymore with the way I wrote the test. Thanks for your vigilance.